### DISCUSSION TITLE
API endpoint for contact points/receivers

### DISCUSSION THREAD

**Discussion by travees** [grafana/grafana#44139] [Alerting]
Any plans to add an API endpoint to manage contact points under unified alerting? 

Based on https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json it appears that most (all?) of the other unified alerting config is supported. It would be great to have support for contact points so that all of unified alerting config could be represented in grafonnet.

Apologies if this issue has already been raised, or this endpoint already exists.

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!
